src/components/AddItem.jsx

```diff
@@ -69,6 +70,26 @@ export default function AddItem() {
         </fieldset>
         <button type="submit">Add Item</button>
       </form>
+
+      <nav className="link-div">
+        <NavLink
+          to="/list"
+          style={({ isActive }) =>
+            isActive ? { fontWeight: 'bold' } : { fontWeight: 'normal' }
+          }
+        >
+          Shopping List
+        </NavLink>
+
+        <NavLink
+          to="/addItem"
+          style={({ isActive }) =>
+            isActive ? { fontWeight: 'bold' } : { fontWeight: 'normal' }
+          }
+        >
+          Add Item
+        </NavLink>
+      </nav>
```

Comment:
I wonder if it would be possible to add this code in a new component "Nav"? That way we can add that to both the AddItem.jsx file and List.jsx? Not sure if you already thought about this and tried or not, but something maybe we can think about as a team later?

Reply:
Oh, that's a great idea, Rachael! I'll post an update here when that feature is implemented and ready for review.

Reply:
Update: See commit below "Created Nav component to contain pre-existing logic." for requested changes. Let us know if you have any questions.

Reply:
Awesome! Thank you!